File: GuardianMonsters/core/src/de/limbusdev/guardianmonsters/services/NullGameStateService.java
package de.limbusdev.guardianmonsters.services;

import de.limbusdev.guardianmonsters.fwmengine.managers.SaveGameManager;
import de.limbusdev.guardianmonsters.model.gamestate.GameState;

public class NullGameStateService implements IGameStateService
{
    @Override
    public void destroy() { }

    @Override
    public SaveGameManager getSaveGameManager() {
        return null;
    }

    @Override
    public GameState getGameState() {
        return null;
    }
}

File: GuardianMonsters/core/src/de/limbusdev/guardianmonsters/services/NullGameStateService.kt
package de.limbusdev.guardianmonsters.services

import de.limbusdev.guardianmonsters.fwmengine.managers.SaveGameManager
import de.limbusdev.guardianmonsters.model.gamestate.GameState

class NullGameStateService : IGameStateService {
    override fun destroy() {}

    override fun getSaveGameManager(): SaveGameManager? {
        return null
    }

    override fun getGameState(): GameState? {
        return null
    }
}
